pilot/pkg/security/authz/builder/extauthz.go

```diff
@@ -189,22 +189,35 @@ func parseService(in *plugin.InputParams, service string, port int) (hostname st
 		err = fmt.Errorf("service must not be empty")
 		return
 	}
-	parts := strings.Split(service, "/")
-	// TODO(yangminzhu): Currently require the user to always specify namespace, may relax this later.
-	if len(parts) != 2 {
-		err = fmt.Errorf("service not in format <namespace>/<hostname>, found: %s", service)
-		return
-	}
 
-	namespace, name := parts[0], parts[1]
-	if svc := in.Push.ServiceIndex.HostnameAndNamespace[host.Name(name)][namespace]; svc != nil {
-		// TODO(yangminzhu): Verify the service and its cluster is supported, e.g. resolution type is not OriginalDst.
-		hostname = string(svc.Hostname)
-		cluster = model.BuildSubsetKey(model.TrafficDirectionOutbound, "", svc.Hostname, port)
-		return
+	// TODO(yangminzhu): Verify the service and its cluster is supported, e.g. resolution type is not OriginalDst.
```

Comment:
not quite follow this comment, mind elaborate?

Reply:
There are some clusters that can not be used with ext-authz, for example, cluster using resolution type OriginalDst. We can log an error instead of using the cluster when this happens.

Reply:
why `OriginalDst` is not supported

Reply:
I think It's an limitation in the ext-authz filter. When ext-authz filter is making the request to the cluster, I don't think it uses anything like iptables or TProxy: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/service_discovery.html#original-destination

Reply:
Original dest implies we are forwarding a request. It doesn't make sense when we are initiating a request, since we do not have any address to send to